moveit_planners/trajopt/src/trajopt_planner_manager.cpp

```diff
@@ -0,0 +1,90 @@
+/*********************************************************************
```

Comment:
These TrajOpt files shouldn't be in this PR: can you retarget for `bullet_trajopt_feature_branch`?